.github/workflows/pr-check.yml

```diff
@@ -0,0 +1,34 @@
+name: PR Check
+
+on:
+  pull_request_target:
+    types: [opened, edited, reopened, synchronize, ready_for_review, review_requested]
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+  statuses: write
+  checks: write
+      
+jobs:
+  check_commit_message:
+    name: Check Commit Message
```

Comment:
Looks like this is checking on ordinary commits? Is that true? We really only want it to check on master, as discussed.

Reply:
That's a question of the configuration of the commit-message-checker bellow. Here it is configured to look at PR titles and body only. And as you can see above the event is pull_request_target (btw we need target here for the labeler or no labels could be added)